libraries/Microsoft.Bot.StreamingExtensions/Microsoft.Bot.StreamingExtensions.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version Condition=" '$(PackageVersion)' == '' ">1.0.0</Version>
+    <Version Condition=" '$(PackageVersion)' != '' ">$(PackageVersion)</Version>
+    <PackageVersion Condition=" '$(PackageVersion)' == '' ">1.0.0</PackageVersion>
+    <PackageVersion Condition=" '$(PackageVersion)' != '' ">$(PackageVersion)</PackageVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>..\..\ChannelSdk\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <PackageId>Microsoft.Bot.StreamingExtensions</PackageId>
+    <Authors>Microsoft</Authors>
+    <Product>Microsoft Bot Protocol SDK</Product>
+    <Company>Microsoft</Company>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>https://github.com/Microsoft/BotBuilder/blob/master/LICENSE</PackageLicenseUrl>
```

Comment:
Most of these should be removed, as they are in our dir.props file. 
[https://github.com/microsoft/botbuilder-dotnet/blob/master/Directory.Build.props](https://github.com/microsoft/botbuilder-dotnet/blob/master/Directory.Build.props)

  <PropertyGroup>
    <Company>Microsoft</Company>

    <!-- Rules found at: [https://aka.ms/Microsoft-NuGet-Compliance](https://aka.ms/Microsoft-NuGet-Compliance) -->

    <!-- Per rules,this must be *EXACTLY* "Microsoft. Otherwise Nuget.org will reject the packages." -->
    <Authors>Microsoft</Authors>
    <Product>Microsoft Bot Builder SDK</Product>
    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
  </PropertyGroup>